tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup.in

```diff
@@ -15,6 +15,9 @@ pdsh -l root -R ssh -S -w $(spacelist_to_commalist $CHROMA_MANAGER ${STORAGE_APP
 # Clean out any yum info, if this is a manual system not automated the underlying repos might have changed.
 yum clean all
 
+# Do upgrade because vagrant doesn't have el7.7 IML#1277
+$PROXY yum upgrade -y
```

Comment:
Should we do:
```
yum -y upgrade --exclude=python2-iml*
```
?

Reply:
There shouldn't  be any IML at this point.

Reply:
Ok i'm fine with this then.